docs/semgrep-supply-chain/malicious-dependencies.md

```diff
@@ -42,9 +38,12 @@ The following table lists the languages for which Supply Chain can detect malici
 
 ## Malicious dependency findings
 
+
 Malicious dependency findings are treated as **critical severity** findings.
 
-If you have set up your Supply Chain policies to block with this condition, malicious dependency findings block a PR or MR in the same way as any other Supply Chain finding.
+If you set up your Supply Chain [policies](https://semgrep.dev/orgs/-/policies/supply-chain) to block critical severity findings, malicious dependency findings block a PR or MR in the same way as any other Supply Chain finding.
+
+From the Supply Chain policies page, you can also configure a policy to trigger conditionally when a dependency `is` or `is not` marked **Malicious**.
```

Comment:
```suggestion
From the Supply Chain policies page, you can also configure a policy to trigger conditionally based on whether a dependency is marked **Malicious**.
```

Not sure if this is the best rewording, but I thought the original was confusing